e2e/flags.go

```diff
@@ -141,9 +169,13 @@ var GKENumNodes = flag.Int("gke-num-nodes", util.EnvInt("GKE_NUM_NODES", Default
 var GKEAutopilot = flag.Bool("gke-autopilot", util.EnvBool("GKE_AUTOPILOT", false),
 	"Whether to create GKE clusters with autopilot enabled.")
 
-// CreateClusters indicates the test framework should create GKE clusters.
-var CreateClusters = flag.Bool("create-clusters", util.EnvBool("CREATE_CLUSTERS", false),
-	"Whether to create GKE clusters, otherwise will assume pre-provisioned GKE clusters.")
+// CreateClusters indicates the test framework should create clusters.
+var CreateClusters = flag.String("create-clusters", util.EnvString("E2E_CREATE_CLUSTERS", CreateClustersDisabled),
+	"Whether to create clusters, otherwise will assume pre-provisioned clusters.")
```

Comment:
Suggest: adding the allowed enum values to the description

Reply:
will do

Reply:
done